scripts/build_coverage.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+cd ./lib
+mvn compile
+cd ..
+cd ./schema
+mvn compile
+cd ..
+cd ./services/port_manager
```

Comment:
Can we use a command to auto go through every subdirectory under service directory so that when we add a new microservice, we don't have to change the script every time.

Reply:
cd services
  for d in ./*/;
  do
      echo "Compile service -  $d"
      cd $d
      mvn clean
      mvn compile
      cd ..
      echo "Compile service -  $d completed"
  done

Reply:
We can do it. Initially I did it, but there was some error on it because it goes to every directory.
I will retry it.